src/commands/hpersist.json

```
@@ -0,0 +1,84 @@
+{
+    "HPERSIST": {
+        "summary": "Remove the existing expiration on a hash key's field(s).",
+        "complexity": "O(1) for each field assigned with TTL, so O(N) to persist N items when the command is called with multiple fields.",
+        "group": "hash",
+        "since": "9.0.0",
+        "arity": -5,
+        "function": "hpersistCommand",
+        "command_flags": [
+            "WRITE",
+            "FAST"
+        ],
+        "acl_categories": [
+            "HASH"
+        ],
+        "key_specs": [
+            {
+                "flags": [
+                    "RW",
+                    "UPDATE"
+                ],
+                "begin_search": {
+                    "index": {
+                        "pos": 1
+                    }
+                },
+                "find_keys": {
+                    "range": {
+                        "lastkey": 0,
+                        "step": 1,
+                        "limit": 0
+                    }
+                }
+            }
+        ],
+        "reply_schema": {
+            "description": "List of values associated with the result of setting expiry on the specific fields, in the same order as they are requested.",
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "oneOf": [
+                    {
+                        "description": "Field does not exist in the provided hash key, or the hash key does not exist",
```

```suggestion
                        "description": "Field does not exist in the provided hash key, or the hash key does not exist.",
```

(I'm not going to tag all instances, but this feels like a bug...)

---

@jsoref , what feels like a bug?

---

The various places that were missing the `.`